lib/routes.js

```diff
@@ -0,0 +1,15 @@
+var routes = {
+  method: "GET",
+  path:   "/version",
+  config: {
+    state: {
+      parse:      true,
+      failAction: "log"
```

Comment:
Why use "log" over the default behavior of returning a Bad Request (400) error response ("error")?